Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:erbium-buster AS build
+ENV LANG C.UTF-8
+
+# Copy data for add-on
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+RUN npm install
+RUN npm run build
+RUN npm prune --production
+
+FROM node:erbium-buster-slim
+COPY --from=build /usr/src/app /usr/src/app
+WORKDIR /usr/src/app
+EXPOSE 3000
+USER root
+CMD [ "node", "dist/bin/server.js", "/dev/ttyUSB0" ]
```

Comment:
This makes no sense to hardcode to this path. People need to get their serial stick into Docker.

Reply:
And also please add a section to the README how to use this Docker file.